reliability-v2/tasks/script/cronjob.sh

```diff
@@ -0,0 +1,155 @@
+#/!/bin/bash
+################################################
+## Author: qili@redhat.com
+## Description: Script for cronjob typ of workload
+## Task: https://issues.redhat.com/browse/OCPQE-13863
+## Bugs: https://issues.redhat.com/browse/OCPBUGS-7962
+## https://issues.redhat.com/browse/OCPBUGS-8217
+## https://issues.redhat.com/browse/OCPBUGS-8439
+## https://issues.redhat.com/browse/OCPBUGS-3652
+## During the test, check the inode and worker node's memory usage on dittybopper's openshift-performance dashboard.
+## Reference: https://docs.openshift.com/container-platform/4.12/nodes/jobs/nodes-nodes-jobs.html#nodes-nodes-jobs-creating-cron_nodes-nodes-jobs
+################################################
+
+USER=${USER:-""}
+image="image-registry.openshift-image-registry.svc:5000/openshift/cli"
+cmd="/bin/sh -c date"
+ns="${USER}-cronjob"
+cronjob_prefix="cronjob"
+
+function _usage {
+    cat <<END
+Usage: $(basename "${0}") [-n <number of cronjobs>] [-s <schedule>] -d
+
```

Comment:
Are separate new empty lines between each option for usage message intended? Otherwise, may just need little bit of cleanup?

Reply:
I cleaned some empty line and just keep one for easily reading.